tests/integration/common.rs

```diff
@@ -138,7 +138,7 @@ impl FarmerBuilder {
         let InnerFarmer { reward_address, n_sectors } = self._build().expect("Infallible");
         let sector_size = subspace_farmer_components::sector::sector_size(
             // TODO: query node for this value
-            1300,
+            35,
```

Comment:
maybe use a constant instead of this magic number? no big deal though, will approve

Reply:
Let's merge it as it is. I'll fix it later. In SDK itself we query runtime, so it will be fine